core/dbt/parser/manifest.py

```diff
@@ -214,15 +212,52 @@ def load(self):
         if self.saved_manifest is not None:
             partial_parsing = PartialParsing(self.saved_manifest, self.manifest.files)
             skip_parsing = partial_parsing.skip_parsing()
-            if not skip_parsing:
+            if skip_parsing:
+                # nothing changed, so we don't need to generate project_parser_files
+                self.manifest = self.saved_manifest
+            else:
                 # create child_map and parent_map
                 self.saved_manifest.build_parent_and_child_maps()
                 # files are different, we need to create a new set of
                 # project_parser_files.
-                project_parser_files = partial_parsing.get_parsing_files()
-                self.partially_parsing = True
-
-            self.manifest = self.saved_manifest
+                try:
+                    project_parser_files = partial_parsing.get_parsing_files()
+                    self.partially_parsing = True
+                    self.manifest = self.saved_manifest
+                except Exception:
+                    # pp_files should still be the full set and manifest is new manifest,
+                    # since get_parsing_files failed
+                    logger.info(
+                        "Partial parsing enabled but an error occurred. Full parsing resumed."
```

Comment:
Really good. Only the word `resumed` is throwing me. How about something like:
```suggestion
                        "Partial parsing enabled but an error occurred. Commencing full re-parse."
```

Reply:
You're right about "resumed" not really worked, but "commencing" doesn't sound quite right to me either so I changed it to "Switching to a full re-parse."

Reply:
Nice, `switching` is the other word I was thinking of for this. I was too taken by [in Star Trek voice] "Computer, commence full re-parse"

Reply:
Hehe.